ClientApp/src/components/admin/common/RecipientTable.tsx

```diff
@@ -72,13 +79,15 @@ const DatatableRecipient: React.FC<Props> = ({
 
   return (
     <Container>
-      {data.map((recipient) => (
-        <Accordion expanded={expanded===recipient}
-          onChange={handleChange(recipient)}
+      {data.map((recipient, index) => (
+        <Accordion
+          expanded={selected == index}
           key={recipient.rowKey}
           className={classes.accordionContainer}
+          onClick={() => { index != selected ? setSelected(index) : setSelected(-1)}}
```

Comment:
Should onClick use handleChange? 

handleChange function is unused after this changes

Reply:
Created a handleSelectedAccordion() function!